src/script/entity/Conversation.ts

```diff
@@ -204,7 +203,6 @@ export class Conversation {
     this.teamId = undefined;
     this.type = ko.observable();
 
-    this.is_loaded = ko.observable(false);
     this.is_pending = ko.observable(false);
```

Comment:
If you've understood what `is_pending` is about, could you document it with JS doc (I'm currently not sure what this state is about)?

Reply:
Added a comment https://github.com/wireapp/wire-webapp/pull/17158/commits/cc00a27937577f5fdf672354e0af6f4c263fbe0c. We set it while messages are being loaded into the conversation' state. With that we later decide if we sometimes abort loading them again if the process is ongoing. I think this could be renamed to `isLoadingMessages`.

Reply:
Yep, this name seems better indeed 👍